Assets/Scripts/CustomExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+public static class CustomExtensions
+{
+    public static void AddToEnemySpawnPointList(this List<List<EnemySpawnPoint>> spawnPointList, EnemySpawnPoint spawnPointToAdd)
+    {
+        try
```

Comment:
You know, if doing this properly, you would check the length of the list and add spawnpoint accordingly :D. Try catch is expensive

Reply:
But it's up to you since this is only run once, I dont care

Reply:
Yeah it just looked cleaner like this :D maybe you're right